designs/lily/src/front.mjs

```diff
@@ -0,0 +1,610 @@
+import { front as titanFront } from '@freesewing/titan'
+import { back } from './back.mjs'
+
+function draftLilyFront({
+  points,
+  Point,
+  paths,
+  Path,
+  measurements,
+  options,
+  complete,
+  paperless,
+  store,
+  macro,
+  utils,
+  snippets,
+  Snippet,
+  sa,
+  absoluteOptions,
+  part,
+}) {
+  /*
+   * Helper method to draw the inseam path
+   */
+  const drawInseam = () =>
+    new Path()
+      .move(points.floorIn)
+      .curve(points.floorInCp2, points.kneeInCp1, points.kneeIn)
+      .curve(points.kneeInCp2, points.forkCp1, points.fork)
+  /*
+   * Helper method to draw the outseam path
+   */
+  const drawOutseam = () => {
+    let waistOut = points.styleWaistOutLily || points.waistOut
+    if (points.waistOut.x < points.seatOut.x) {
```

Comment:
A ternary return front-loads the cognitive overload:
```js
return (points.waistOut.x < points.seatOut.x)
  ? new Path()
      .move(waistOut)
      .curve(points.seatOut, points.kneeOutCp1, points.kneeOut)
      .curve(points.kneeOutCp2, points.floorOutCp2, points.floorOut)
  :  new Path()
      .move(waistOut)
      ._curve(points.seatOutCp1, points.seatOut)
      .curve(points.seatOutCp2, points.kneeOutCp1, points.kneeOut)
      .curve(points.kneeOutCp2, points.floorOutCp2, points.floorOut)
```

Reply:
One day, I'll learn to use these without being told. :)